doxia-integration-tools/src/main/java/org/apache/maven/doxia/tools/DefaultSiteTool.java

```diff
@@ -128,36 +129,46 @@ public class DefaultSiteTool implements SiteTool {
     // Public methods
     // ----------------------------------------------------------------------
 
+    /** {@inheritDoc} */
     public Artifact getSkinArtifactFromRepository(
-            ArtifactRepository localRepository,
-            List<ArtifactRepository> remoteArtifactRepositories,
-            DecorationModel decoration)
+            RepositorySystemSession repoSession, List<ArtifactRepository> remoteArtifactRepositories, Skin skin)
```

Comment:
Why not - `List<RemoteRepository>` - we have it on `MavenProject#getRemoteProjectRepositories`

Reply:
Very good question. So this is basically old vs. new, right? Previously, in Maven Core namespace, now in Resolver. Looking at the impls it does use `RepositoryUtils`.